macros.tex

```diff
@@ -40,17 +40,20 @@
 % General macro for extracting the jobnumber from the jobid
 \macro{\jobnumber}{\BeforeSubString{.}{\jobid}}
 \macro{\pbsserver}{\BehindSubString{.}{\jobid}}
-% Macro for winscp/putty fingerprint allert
-\macro{\firsttimeconnection}{
-      The first time you make a connection to the login node, a Security
-      Alert will appear and you will be asked to verify the authenticity of the
-      login node.
-      \ifgent
-      Make sure the fingerprint in the Alert matches \strong{\loginfingerprint}
-      \fi
+% Macro for winscp/putty fingerprint alert
+\newcommand{\firsttimeconnection}{
+The first time you make a connection to the login node, a Security
+Alert will appear and you will be asked to verify the authenticity of the
+login node.
+\ifgent
+Make sure the fingerprint in the alert matches one of the following:
+\puttyFirstConnect
+If it does, press \strong{\emph{Yes}}, if it doesn't, please contact \hpcinfo
+\else
+Now press \strong{\emph{Yes}}
```

Comment:
Wait, this is the bit used for non-UGent sites? Don't we have host keys for others sites too?
We need to be a bit more careful here, just telling non-UGent users to blindly accept is not a good idea...

Reply:
Well, that's what is currently happening (see http://hpcugent.github.io/vsc_user_docs/pdf/intro-HPC-windows-brussel.pdf#section.2.2). I wanted to preserve how it was, but I'll update the other sites if you say so.

Reply:
Well, if that's the case, then just open an issue to clarify that the other sites should look into this, since it's not OK imho... I'll then reach out to the relevant people.

Reply:
Done #228

Reply:
we have the fingerprints for all sites, this should be fixed

Reply:
if you don't then just check them ;) see the comment in the ugent file on how to do this